system-int-test/test/cmr/system_int_test/ingest/granule_ingest_test.clj

```diff
@@ -333,28 +333,31 @@
     (is (= "Name/With/Slashes" (:native-id ingested-concept)))))
 
 (deftest granule-schema-validation-test
-  (are [concept-format validation-errors]
-       (let [collection (d/ingest-umm-spec-collection "PROV1" (data-umm-c/collection {}))
-             umm-granule (dg/granule-with-umm-spec-collection collection (:concept-id collection) {:native-id "Name/With/Slashes"})
-             concept (d/item->concept
-                       (dg/granule-with-umm-spec-collection collection (:concept-id collection) {:beginning-date-time "2010-12-12T12:00:00Z"})
-                       concept-format)
-             {:keys [status errors]}
-             (ingest/ingest-concept
-               (assoc concept
-                      :format (mt/format->mime-type concept-format)
-                      :metadata (-> concept
-                                    :metadata
-                                    (str/replace "2010-12-12T12:00:00" "A")
-                                    ;; this is to cause validation error for iso-smap format
-                                    (str/replace "gmd:DS_Series" "XXXX"))))]
-         (index/wait-until-indexed)
-         (= [400 validation-errors] [status errors]))
-
-       :echo10 ["Line 1 - cvc-datatype-valid.1.2.1: 'A.000Z' is not a valid value for 'dateTime'."
-                "Line 1 - cvc-type.3.1.3: The value 'A.000Z' of element 'BeginningDateTime' is not valid."]
-
-       :iso-smap ["Line 1 - cvc-elt.1: Cannot find the declaration of element 'XXXX'."]))
+  (are3 [concept-format validation-errors]
+    (let [collection (d/ingest-umm-spec-collection "PROV1" (data-umm-c/collection {}))
+          concept (d/item->concept
+                   (dg/granule-with-umm-spec-collection collection (:concept-id collection) {:beginning-date-time "2010-12-12T12:00:00Z"})
+                   concept-format)
+          invalid-granule (update concept :metadata
+                                  #(-> %
```

Comment:
This is one of the few times where I think the shorthand makes sense for a multi-line function.